exampleSite/utilities/README.md

```diff
@@ -59,6 +66,8 @@ Website = ""
 Facebook = ""
 Linkedin = ""
 Pronouns = ""
+Github = ""
+Twitter = ""
```

Comment:
Sure. It's cool to put it here I guess. I just don't want to overwhelm people with decisions when they first start.